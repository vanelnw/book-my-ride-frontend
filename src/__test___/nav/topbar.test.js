import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { BrowserRouter as Router } from 'react-router-dom';

import TopNav from '../../components/nav/Topbar';
import store from '../../redux/store';

it('Cars renders correctly', () => {
  const tree = renderer.create(
    <Provider store={store}>
      <Router>
        <TopNav />
      </Router>
    </Provider>,
  );
  expect(tree).toMatchSnapshot();
});
