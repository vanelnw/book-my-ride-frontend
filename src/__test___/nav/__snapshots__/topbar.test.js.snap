// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cars renders correctly 1`] = `
<div
  className="topnav"
>
  <div
    className="logo-cont"
  >
    <img
      alt="logo"
      className="logo"
      src="logo.png"
    />
  </div>
  <div
    className="auth"
  >
    <button
      className="button"
      type="button"
    >
      <a
        className="login"
        href="/login"
        onClick={[Function]}
      >
        Login
      </a>
    </button>
    <button
      className="button"
      type="button"
    >
      <a
        className="signup"
        href="/register"
        onClick={[Function]}
      >
        Register
      </a>
    </button>
  </div>
</div>
`;
